test/entrypoint.sh

```diff
@@ -30,6 +30,13 @@ MYSQL_CONTAINER=1 $DIR/create_db.sh
 # Set up rabbitmq exchange
 rabbitmq-setup -server amqp://boulder-rabbitmq
 
+# Make sure SoftHSM container is up and using the correct key
+wait_tcp_port boulder-hsm 5657
+if [ ! -f test/test-ca.key.der ]; then
+    openssl rsa -in test/test-ca.key -inform pem -out test/test-ca.key.der -outform der
+fi
+PKCS11_PROXY_SOCKET="tcp://boulder-hsm:5657" pkcs11-tool --module=/usr/local/lib/libpkcs11-proxy.so --write-object test/test-ca.key.der --type privkey --label key_label --pin 5678 --login --so-pin 1234
```

Comment:
Alright, I'm willing to go with this for now. Maybe we'll want to tweak it later.